nowcasting_dataset/config/model.py

```diff
@@ -161,13 +161,13 @@ def check_start_and_end_datetime(cls, values):
 class PV(DataSourceMixin, StartEndDatetimeMixin):
     """PV configuration model"""
 
-    pv_filename: str = Field(
-        "gs://solar-pv-nowcasting-data/PV/PVOutput.org/UK_PV_timeseries_batch.nc",
-        description=("The NetCDF file holding the solar PV power timeseries."),
+    pv_filenames: List = Field(
```

Comment:
tiny tweak: [As of Python 3.9, we can use `list` and `tuple` instead of `typing.List` and `typing.Tuple`](https://stackoverflow.com/a/39458225/732596).

```suggestion
    pv_filenames: list = Field(
```